index.html

```diff
@@ -62,8 +62,11 @@
             <li><a href="#book" class="st-research-num st-scroll-to">Book manuscript</a></li>
             <li><a href="#research" class="st-research-num st-scroll-to">Research</a></li>
             <li><a href="#education" class="st-research-num st-scroll-to">Education</a></li>
```

Comment:
Forgot to change the name of the link to the section "cv". The scroll won't work this way.